test/test/otlp/OtlpTemporalityTest.java

```diff
@@ -0,0 +1,52 @@
+package test.otlp;
+
+import one.profiler.AsyncProfiler;
+import one.profiler.Events;
+import one.profiler.Counter;
+import java.nio.ByteBuffer;
+import io.opentelemetry.proto.profiles.v1development.*;
+
+public class OtlpTemporalityTest {
+
+    public static void main(String[] args) throws Exception {
+        AsyncProfiler profiler = AsyncProfiler.getInstance();
+        profiler.start(Events.CPU, 1_000_000);
+
+        ValueType sampleType = getSampleType(profiler);
+        assert sampleType.getAggregationTemporality() == AggregationTemporality.AGGREGATION_TEMPORALITY_DELTA : 
+            "First dump should be DELTA";
```

Comment:
We don't need the assertion error messages anymore, since we're not using the for loop anymore.

```suggestion
        assert getSampleType(profiler).getAggregationTemporality() == AggregationTemporality.AGGREGATION_TEMPORALITY_DELTA;
```